pallets/payment-streams/src/tests.rs

```diff
@@ -99,21 +100,28 @@ mod create_stream {
             let alice: AccountId = 0;
             let bob: AccountId = 1;
 
-            // Register Alice as a BSP with 100 units of data
+            // Register Alice as a BSP with 100 units of data and get her BSP ID
             register_account_as_bsp(alice, 100);
+            let alice_bsp_id =
+                <StorageProviders as ProvidersInterface>::get_provider_id(alice).unwrap();
 
             // Create a payment stream from Bob to Alice of 10 units per block
             let rate: BalanceOf<Test> = 10;
             assert_ok!(PaymentStreams::create_payment_stream(
                 RuntimeOrigin::root(),
-                alice,
+                alice_bsp_id,
```

Comment:
So now every BSP has to keep track of their provider id in order to execute these extrinsics? I think we should be using the base account id explicitely for extrinsics and storage related queries that would not necessitate users or providers having to keep  a mapping of this locally or having to pre query storage to get the provider id before executing something.

Reply:
Nope! It is a bit confusing so I wrote it on the docs of each one, but the extrinsics of this pallet (except `charge_payment_stream`) are not to be actually used in prod (the `PaymentStreamsInterface` should be used by another pallet instead), instead they are designed to help with manual testing in scenarios such as when trying the chain in polkadot{.js}.

This is why they have as a parameter the `sp_id` instead of the `sp_account`, because for the functionality of creating, updating or deleting a payment stream to be executed, the file system pallet should call the payment streams pallet with its interface when a SP's request to store is accepted or a user executes a file deletion